docs/source/posts/2021/2021-06-21-week-3-sajag.rst

```diff
@@ -0,0 +1,45 @@
+Second week of coding!
+======================
+
+.. post:: June 21 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the third weekly check-in. I'll be sharing my progress for the second week of coding.
+
+What did you do this week?
+--------------------------
+I created an example to demonstrate how one can render multiple bonds (double and triple). This required me to write an algorithm to detect bonding.
+I used `this <https://www.kaggle.com/aekoch95/bonds-from-structure-data>`_ as a reference and made a few tweaks of my own to detect the presence of double/triple bonds from interatomic distances.
+The math involved in generating the coordinates of bonds was quite intriguing. Preview (molecules rendered: Ethane, Ethene, Ethyne from left to right) - 
```

Comment:
same as above, use caption and not this preview sentence

Reply:
done